components/Hero/src/index.scss

```diff
@@ -147,6 +147,95 @@
   padding-block-end: var(--denhaag-hero-theme-page-title-padding-block-end);
 }
 
+/* added .denhaag-hero to --them-image, otherwise need to use !important to overwrite */
+.denhaag-hero.denhaag-hero--theme-image {
+  background-color: var(--denhaag-hero-theme-image-background-color);
+  background-image: var(--denhaag-hero-theme-image-background-image);
+}
+
+.denhaag-hero--theme-image .denhaag-hero__container {
+  --denhaag-hero-container-max-width: var(--denhaag-hero-theme-image-container-max-width);
+
+  padding-block-start: var(--denhaag-hero-theme-image-container-padding-block-start);
+  display: var(--denhaag-hero-theme-image-container-display);
+  margin-inline-end: var(--denhaag-hero-theme-image-container-margin-inline);
+  margin-inline-start: var(--denhaag-hero-theme-image-container-margin-inline);
+  padding-inline-end: var(--denhaag-hero-theme-image-container-padding-inline);
+  padding-inline-start: var(--denhaag-hero-theme-image-container-padding-inline);
+
+  @media (min-width: 768px) {
+    --denhaag-hero-container-max-width: var(--denhaag-hero-theme-image-container-max-width-tablet);
```

Comment:
Zo te zien wordt er al `-m, `-l` en `-xl` gebruikt in deze file, graag hierin één lijn in houden ( https://github.com/nl-design-system/denhaag/blob/795d0bd806af725964215d7bafc8aed33e76d5bd/components/Hero/src/index.scss#L378 )